db/migrate/20181016140921_migrate_orch_stacks_to_have_ownership_concept.rb

```diff
@@ -0,0 +1,85 @@
+class MigrateOrchStacksToHaveOwnershipConcept < ActiveRecord::Migration[5.0]
+  class ExtManagementSystem < ApplicationRecord
```

Comment:
@d-m-u 
Migration have to be independent from our app as much as possible - so it is recommended to use `ActiveRecord::Base` instead of `ApplicationRecord`.

see https://github.com/ManageIQ/manageiq/pull/7312/files#r56356275

thanks